src/main/scala/org/broadinstitute/dsde/vault/model/Boss.scala

```diff
@@ -41,11 +41,13 @@ object BossDefaults {
   val storagePlatform = VaultConfig.BOSS.defaultStoragePlatform
   val validityPeriodSeconds = VaultConfig.BOSS.defaultValidityPeriodSeconds
 
-  def getCreationRequest(fpath: String, forceLocation: Option[String]): BossCreationObject = forceLocation match {
-    case Some(forceLocation) if forceLocation.toBoolean =>
-      new BossCreationObject(fpath, storagePlatform, 0, ownerId, readers, writers, Option.empty, Some(fpath), Some(true))
-    case _ =>
-      new BossCreationObject(fpath, storagePlatform, 0, ownerId, readers, writers)
+  def getCreationRequest(fpath: String, forceLocation: Option[String]): BossCreationObject = {
+    forceLocation.getOrElse("false").toBoolean match {
+      case true =>
+        new BossCreationObject(fpath, storagePlatform, 0, ownerId, readers, writers, Option.empty, Some(fpath), Some(true))
```

Comment:
I don't know under what scenario it's possible for `fpath` to be `null` here, but if it ever happens then `Some(fpath)` will result in the weird `Some(null)`.  `Option(fpath)` will give `None` in that case, so it's safer.